package.json

```diff
@@ -100,6 +100,7 @@
     "bem-cn-lite": "3.0.0",
     "blueimp-file-upload": "9.22.1",
     "body-parser": "1.18.3",
+    "caniuse-lite": "^1.0.30001165",
```

Comment:
Sorry I thought I commented on 👆🏽. is this needed? (also the `.ruby-version` file)

Reply:
thanks for catchint that! removed this change and the `.ruby-version` file but do I also need to remove the changes in the `yarn.lock` file too?

Reply:
Yes I believe so. Or you can use yarn to remove it. I am guessing a `yarn add caniuse-lite` was accidentally run in the force directory, adding the library to `package.json` and `yarn.lock` so `yarn remove caniuse-lite` would remove it.